### PR TITLE
foreman-puppet: don't run db:migrate in postinst

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: plugins/ruby-foreman-puppet/debian/postinst [hunk 1]
```diff
@@ -38,19 +38,6 @@ else
...
-# DB migrate/seed after install
-if [ -f /usr/share/foreman/config/database.yml ]; then
-  if [ ! -z "${DEBUG}" ]; then
-    /usr/sbin/foreman-rake db:migrate || true
-    /usr/sbin/foreman-rake db:seed || true
-    /usr/sbin/foreman-rake apipie:cache:index || true
-  else
-    /usr/sbin/foreman-rake db:migrate >> $LOGFILE 2>&1 || true
-    /usr/sbin/foreman-rake db:seed >> $LOGFILE 2>&1 || true
-    /usr/sbin/foreman-rake apipie:cache:index >> $LOGFILE 2>&1 || true
```
Comment (evgeni) [inline plugins/ruby-foreman-puppet/debian/postinst:50]: huh, I guess *this* should still happen?

Comment (ehelms) [reply plugins/ruby-foreman-puppet/debian/postinst:50]: I do think we should keep, I wondered if this should be the plugin version of it like RPM?

https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/foreman/foreman/foreman.spec#L911

Comment (evgeni) [reply plugins/ruby-foreman-puppet/debian/postinst:50]: yeah (and no), the line you're linking to is the compile step, not the index step

**Comment by evgeni** [comment]
yepp, this is def needed to unbreak upgrades

**Review by ehelms — APPROVED** [review]

**Pull request merged**